widgets/xbacklight_linux.lua

```diff
@@ -0,0 +1,31 @@
+---------------------------------------------------
+-- Licensed under the GNU General Public License v2
+--  * (c) 2019, Daan V. <daan@dirkvanoverloop.be>
+---------------------------------------------------
+
+-- {{{ Grab environment
+local tonumber = tonumber
+local math = { floor = math.floor }
+local type = type
+local setmetatable = setmetatable
```

Comment:
`setmetatable` is no longer used--using `luacheck` can help you avoid this

Reply:
I didn't know about luacheck, that's quite a useful tool.